crates/telio-batcher/src/batcher.rs

```diff
@@ -37,56 +50,76 @@ where
         Self {
             actions: HashMap::new(),
             notify_add: tokio::sync::Notify::new(),
```

Comment:
I personally like not to import stuff that's only used once in the file, it simplifies things since there's less to think about what we import and what we use from the imports. In this case there's no import of `Notify` so it's clear it might be only occasionally used.

Of course, this is more of a matter of discipline, I could use `tokio::sync::Notify` multiple times no problem :)